e2e/test/scenarios/dashboard-filters/dashboard-filters-auto-apply.cy.spec.js

```diff
@@ -113,22 +120,25 @@ describe("scenarios > dashboards > filters > auto apply", () => {
       cy.wait("@updateDashboard");
       cy.findByLabelText("Auto-apply filters").should("be.checked");
     });
-    filterWidget().within(() => {
-      cy.findByText("2 selections").should("be.visible");
-      cy.get("@cardQuery.all").should("have.length", 3);
-    });
+    filterWidget().findByText("Widget").should("be.visible");
+    getDashboardCard().findByText("Rows 1-4 of 54").should("be.visible");
+    cy.get("@cardQuery.all").should("have.length", 4);
 
-    // last applied parameter values should be used when disabling auto applying filters,
-    // even if previously there were draft parameter values
+    cy.log(
```

Comment:
This test tests that the previously unapplied filters won't be used when disabling dashboard auto-apply filters. But since we just changed it so that we never have **unapplied filters** when turning the dashboard auto-apply filters back on, I have to modify this test by making sure the last auto-applied filters (the filters we modified when `auto_apply_filters: true`) are used instead of the filters applied from unapplied filters (the last filters we modified when `auto_apply_filters: false`, which were applied automatically once we turned the auto-apply filters back on).